src/Microsoft.TemplateEngine.Core/CommonOperators.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Core.Expressions;
+
+namespace Microsoft.TemplateEngine.Core
+{
+    public enum Operators
+    {
```

Comment:
This should have a "None" defined (or some other name with similar semantic meaning).